src/node_contextify.cc

```diff
@@ -934,6 +934,30 @@ Maybe<bool> StoreCodeCacheResult(
   return Just(true);
 }
 
+// TODO(RaisinTen): Reuse in ContextifyContext::CompileFunction().
+MaybeLocal<Function> CompileFunction(Isolate* isolate,
+                                     Local<Context> context,
+                                     Local<String> filename,
+                                     Local<String> content,
+                                     std::vector<Local<String>>* parameters) {
+  ScriptOrigin script_origin(isolate, filename, 0, 0, true);
+  ScriptCompiler::Source script_source(content, script_origin);
+
+  Local<Function> fn;
+  if (!ScriptCompiler::CompileFunction(context,
+                                       &script_source,
+                                       parameters->size(),
+                                       parameters->data(),
+                                       0,
+                                       nullptr,
+                                       ScriptCompiler::kEagerCompile)
+           .ToLocal(&fn)) {
+    return MaybeLocal<Function>();
+  }
+
+  return fn;
```

Comment:
```suggestion
  return ScriptCompiler::CompileFunction(context,
                                       &script_source,
                                       parameters->size(),
                                       parameters->data(),
                                       0,
                                       nullptr,
                                       ScriptCompiler::kEagerCompile);
```

Does this work, since ScriptCompiler::CompileFunction already returns `MaybeLocal<Function>`?